app/models/builders/manual_builder.rb

```diff
@@ -24,10 +18,14 @@ def call(attrs)
       use_originally_published_at_for_public_timestamp: true,
     }
 
-    factory.call(default_attrs.merge(attrs))
+    attrs = default_attrs.merge(attrs)
```

Comment:
Minor: I'd have a slight preference for not re-assigning `attrs` here which I find a bit confusing, i.e. in-line this local variable into the call to `Manual.new` on the next line.